contrib/babelfishpg_tsql/src/pltsql_ruleutils.h

```diff
@@ -0,0 +1,12 @@
+/*
+ *
+ * pltsql_ruleutils.h
+ *	  Declarations for pltsql_ruleutils.h
+ *
+ */
+
+#ifndef PLTSQL_RULEUTILS_H
+#define PLTSQL_RULEUTILS_H
+
+#endif							/* PLTSQL_RULEUTILS_H */
+
```

Comment:
What is the use of this file?

Reply:
Removed this file in the commit. It's not needed.